wremnants/muon_efficiencies.py

```diff
@@ -124,13 +162,30 @@ def make_muon_efficiency_helpers(filename = data_dir + "/testMuonSF/scaleFactorP
         axis_pt_eff_tensor = hist.axis.Regular(nptbins, axis_pt_eff.edges[0], axis_pt_eff.edges[nptbins], name = axis_pt_eff.name, overflow = False, underflow = False)
     elif isinstance(axis_pt_eff, bh.axis.Variable):
         axis_pt_eff_tensor = hist.axis.Variable(axis_pt_eff.edges[:nptbins+1], name = axis_pt_eff.name, overflow = False, underflow = False)
+    # repeat for tracking
+    if isinstance(axis_pt_eff_tracking, bh.axis.Regular):
```

Comment:
I think you could make this a bit simpler by assigning the function to a type and/or reusing common args. As it stands it can be hard to work out what's actually changing

e.g. 

common_args = {"flow" : True, "name" : axis_pt_eff_tracking.name}
hist.axis.Variable(axis_pt_eff_tracking.edges[:nptbins_tracking+1], **common_args)

Reply:
I think one still wants to preserve the regular or variable axis, so the if-elif is still needed to choose it. I would let Josh comment, for the common args ok, but this is adding one line per axis per histogram, the gain is questionable

Reply:
I think it would be more clear eventually but it's fine for now.

Reply:
Let's keep it for now